src/frontend/src/eth/services/erc20.services.ts

```diff
@@ -116,6 +117,9 @@ const loadErc20UserTokens = async (params: {
 							version: fromNullable(version),
 							enabled: fromNullable(enabled) ?? true
 						},
+						// 1. TODO(GIX-2740): check uf user token is actually a match in the environment static metadata
+						// +
+						// 2. TODO(GIX-2740): check if metadata for address already loaded and reuse - using Infura is not a certified call anyway
```

Comment:
here you mean to cache what we load? I imagine for a specific period of time, to guarantee that any change in metadata is updated?

Reply:
I just mean check if metadata is already loaded in store. Added the keyword "store".